### ISSUE TITLE
Duplicates in BVCurationsUICollectionView

### ISSUE THREAD

**Issue by rwbutler** [bazaarvoice/bv-ios-sdk#27]
## The Problem

When scrolling through a collection view duplicate items show in the feed.

## Steps to Reproduce

Load any feed into the collection view and scroll until new cells are loaded in from the data source. Subsequent data fetches (following the initial original fetch) request data from `_lastFetchedTimeStamp`. Data returned as part of this request will include items with this exact timestamp meaning that the last item fetched will be the same as the first item in the subsequent fetch. This causes items to be repeated in the feed and thus in the collection view.

## Expected Behavior

There should not be duplicates cells in the collection view.

## Environment

* Specific BVSDK submodule that exhibits the issue:
* BVSDK version (or git revision) that exhibits the issue: 6.9.0
* Last BVSDK version that did not exhibit the issue (if applicable):
* iOS version that exhibits the issue: 12.1
* Last iOS version that exhibits the issue (if applicable):
* Xcode version: 10.1
* iOS Mobile SDK version build for under test:
* Real device or simulator: All

## Details

As a quick hack, the following in `loadCurationsFeed` will resolve the issue (although I'm sure there's a more elegant solution to this involving fetching items *after* the `_lastFetchedTimeStamp` and not including):

```objective_c
            // Filter items with the same timestamp as the last fetched item i.e. duplicates.
                                NSMutableArray<BVCurationsFeedItem *>* filteredItems = [items mutableCopy];
                                if (filteredItems.count > 0 && filteredItems.firstObject.timestamp == self->_lastFetchedTimeStamp) {
                                    [filteredItems removeObject:filteredItems.firstObject];
                                }
                                [BVPixel trackEvent:inViewEvent];
                                
                                weakSelf.allItemsFetched = !filteredItems.count;
                                weakSelf.pendingUpdate = NO;
                                weakSelf.lastFetchedTimeStamp = filteredItems.lastObject.timestamp;
                                [weakSelf.curationsFeedItems addObjectsFromArray:filteredItems];
                                [weakSelf
                                 updateInfiniteScrollMeasurements:weakSelf.curationsFeedItems.count
                                 itemsPerRow:weakSelf.itemsPerRow];
                                
                                BOOL errorState = filteredItems.count > limit; // if for whatever reason the
                                // api returns more items than
                                // expected we'll just reload
```

## Link to BVSDK Logs

Create a [GIST](https://gist.github.com) which is a paste of your _full_ logs, and link them here.
Do _NOT_ paste your full logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include logs!

## Code To Reproduce Issue [ Very Good To Have ]

Please remember that with sample code it's easier to reproduce the bug and it's much faster to fix it.

Creating a [GIST](https://gist.github.com) with the sample is good but attaching a link to a compilable/runnable example is _better_.

**Comment by cameron-ollivierre** [comment]
Thanks for reporting this issue . We've opened an internal ticket to investigate and update the sdk with a fix.

**Comment by cameron-ollivierre** [comment]
We released 8.0.5 with a fix for this issue. Please see https://github.com/bazaarvoice/bv-ios-sdk/commit/a03798915fc956dfaf04a0d3888ae128d6a426e8 for details

**Comment by rwbutler** [comment]
That's great, thanks for fixing this 👍🏻
